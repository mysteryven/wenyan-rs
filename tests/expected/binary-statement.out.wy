3
3
-1
1
20
20
