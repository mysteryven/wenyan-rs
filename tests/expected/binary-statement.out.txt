3
3
-1
1
20
20
一二
三四
